README.md

```
@@ -14,44 +14,69 @@ app/
+    this file and edit with your Sentry server values if you'd like to enable crash reporting.
+12. **_Voila_** 🎉 You're now ready to run Tari Universe Wallet! Please follow the next item on how to setup your emulator, or you can already run it
+    on your device.
 
 ### Emulator Configuration
 
-Aurora Android native libraries only support `armeabi-v7a`, `arm64-v8a` and `x86_64` [ABIs](https://developer.android.com/ndk/guides/abis). Therefore you can only run Aurora on devices and emulators with a supported ABI. Android Studio emulators use the unsupported `x86` ABI as default, but you can download an `x86_64` image for your emulator and run Aurora on it. Please `Download` an `x86_64` image under the `x86 Images` tab in the `System Image` step of `Virtual Device Configuration.` <p align="center"><img src="https://raw.githubusercontent.com/tari-project/wallet-android/development/readme-files/05_x86_64.png"></p>
+Tari Universe Wallet Android native libraries only support `armeabi-v7a`, `arm64-v8a` and
+`x86_64` [ABIs](https://developer.android.com/ndk/guides/abis). Therefore you can only run Tari Universe Wallet on devices and emulators with a
+supported ABI. Android Studio emulators use the unsupported `x86` ABI as default, but you can download an `x86_64` image for your emulator and run
+Aurora on it. Please `Download` an `x86_64` image under the `x86 Images` tab in the `System Image` step of
+`Virtual Device Configuration.` <p align="center"><img src="https://raw.githubusercontent.com/tari-project/wallet-android/development/readme-files/05_x86_64.png"></p>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)